x/sync/client.go

```diff
@@ -261,12 +260,9 @@ func (c *client) get(ctx context.Context, request []byte) (ids.NodeID, []byte, e
 	}
 	if err != nil {
 		c.metrics.RequestFailed()
-		c.networkClient.TrackBandwidth(nodeID, 0)
```

Comment:
Pretty sure this was broken previously when `RequestAny` returned an error (as we do not guarantee that `nodeID` is populated correctly)